tools/sctool/src/main/java/com/horizen/ScBootstrappingTool.java

```diff
@@ -1,10 +1,15 @@
 package com.horizen;
 import java.io.File;
 import java.util.Scanner;
+import java.util.Set;
+import java.util.TreeSet;
+
+import com.horizen.cryptolibprovider.CryptoLibProvider;
```

Comment:
These imports are unused. Can be removed.

Reply:
agree. Revert the file to previous version

Reply:
ok, done